storage/Utils/AppUtil.h

```diff
@@ -34,6 +34,7 @@
 #include <map>
 #include <chrono>
 
+#undef _
 
```

Comment:
Try to replace the #include "storage/Utils/XmlFile.h" by #include <libxml/tree.h> in Arch.h.